doc/spec_exceptionHandling.dox

```diff
@@ -49,48 +51,50 @@ namespace ChimeraTK {
+    - \anchor exceptionHandling_b_3_2_1 3.2.1 In a recovery group, DeviceManagers wait until all involved DeviceManagers successfully complete the open step before starting the initialisation handler in B.3.2.2 ("barrier POST-OPEN"). [\ref testExceptionHandling_b_3_2_1 "T"]
+    - \anchor exceptionHandling_b_3_2_2 3.2.2 The so-called initialisation handlers are executed (see \ref exceptionHandling_b_3_3 "3.3") [\ref testExceptionHandling_b_3_2_2 "T"].
+      - \anchor exceptionHandling_b_3_2_2_1 3.2.2.1 The device is closed before the initialisation handler is called, and reopened afterwards. This allows devices which can only be opened once to use external init handler scripts. \ref exceptionHandling_comment_b_3_2_2_1 "(*)" [\ref testExceptionHandling_b_3_2_2_1 "T"]
+      - \anchor exceptionHandling_b_3_2_2_2 3.2.2.2 In a recovery group, DeviceManagers wait until all involved DeviceManagers complete the initialisation handler step before restoring register values in B.3.2.3 ("barrier POST-INIT-HANDLER"). [\ref testExceptionHandling_b_3_2_2_2 "T"]
+      - \anchor exceptionHandling_b_3_2_2_3 3.2.2.3 If any DeviceManager sees an exception in one of its initialisation handlers, *all* DeviceManagers in the recovery group restart the recovery procedure after the POST-INIT-HANDLER barrier. [\ref testExceptionHandling_b_3_2_2_3 "T"]
+    - \anchor exceptionHandling_b_3_2_3 3.2.3 All registers that have been written since the start of the application are restored with their latest values. The register values are restored in the same order they were written. Registers of the type ChimeraTK::Void are not written. \ref comment_b_3_2_3 "(*)" [\ref testExceptionHandling_b_3_2_3 "T"]
+      - \anchor exceptionHandling_b_3_2_3_1 3.2.3.1 In a recovery group, DeviceManagers wait until all involved DeviceManagers complete the register value restoring before clearing the internal error flag in B.3.2.4 ("barrier POST-WRITE-RECOVERY"). [\ref testExceptionHandling_b_3_2_3_1 "T"]
+      - \anchor exceptionHandling_b_3_2_3_2 3.2.3.2 If any DeviceManager sees an exception while restoring register values, *all* DeviceManagers in the recovery group restart the recovery procedure after the POST-WRITE-RECOVERY barrier. [\ref testExceptionHandling_b_3_2_3_2 "T"]
+    - \anchor exceptionHandling_b_3_2_4 3.2.4 The DeviceManager clears the internal error flag. From now on, exceptions reported via DeviceManager::reportException() will trigger a new recovery process.
+      - \anchor exceptionHandling_b_3_2_4_1 3.2.4_1 In a recovery group, DeviceManagers wait until all involved DeviceManagers clear their internal error flag before activating the asynchronous read in B.3.2.5 ("barrier POST-CLEAR-ERROR"). [\ref testExceptionHandling_b_3_2_4_1 "T"]
```

Comment:
Typo (_ instead of .):
```suggestion
      - \anchor exceptionHandling_b_3_2_4_1 3.2.4.1 In a recovery group, DeviceManagers wait until all involved DeviceManagers clear their internal error flag before activating the asynchronous read in B.3.2.5 ("barrier POST-CLEAR-ERROR"). [\ref testExceptionHandling_b_3_2_4_1 "T"]
```
Also, I think a footnote explaining why this additional barrier was necessary will help in future...